ArduPlane/Log.cpp

```diff
@@ -330,6 +366,15 @@ const struct LogStructure Plane::log_structure[] = {
       "PIQY", PID_FMT,  PID_LABELS, PID_UNITS, PID_MULTS },
     { LOG_PIQA_MSG, sizeof(log_PID),
       "PIQA", PID_FMT,  PID_LABELS, PID_UNITS, PID_MULTS },
+
+// @LoggerMessage: AETR
+// @Description: Value received by control surfaces of the aircraft
+// @Field: TimeUS: Microseconds since system startup
+// @Field: Ail: Output sent to the Aileron (between -4500 to 4500)
+// @Field: Elev: Output sent to the Elevator (between -4500 to 4500)
+// @Field: Thr: Output sent to the Throttle (between -4500 to 4500)
+// @Field: Rudd: Output sent to the Rudder (between -4500 to 4500)
+// @Field: Flap: Output sent to the Flap (between -4500 to 4500)
```

Comment:
Usually one has multiple flaps :-)
```suggestion
// @Field: Flaps: Output sent to the Flap (between -4500 to 4500)
```

Reply:
The field's name is Flap and therefore I kept it the same. I have changed the field's description to write flaps.